src/MissionCreator.sol

```diff
@@ -142,6 +147,7 @@ contract MissionCreator is Ownable, IERC721Receiver {
             lockedUntil: 0, // The split is not locked, meaning the project owner can remove it or change it at any time.
```

Comment:
Should it be locked? Or do we ever change this?

Reply:
done